src/main/java/com/codecool/barbershop/barbershop/dashboard/DashboardController.java

```diff
@@ -0,0 +1,26 @@
+package com.codecool.barbershop.barbershop.dashboard;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@CrossOrigin("*")
+@RequestMapping("api/v1/dashboard")
```

Comment:
Great that you're versioning the api.  
It would be good to document it. Then potential users of this api will know what endpoints are in version 1 which are in version 2, and so on.